app/admin/admin.rb

```diff
@@ -24,10 +40,14 @@
 
   form do |f|
     f.inputs "User Details" do
+      f.semantic_errors *f.object.errors.keys
```

Comment:
oh, I see what it does. 

```ruby
def func1 *aaa
  pp aaa
end
```
`func1 [4,3,2,2]`  -> [[4, 3, 2, 2]]
`func1 *[4,3,2,2]` -> [4, 3, 2, 2]
`func1 4,3,2,2` -> [4, 3, 2, 2]
`func1 4` -> [4]
`func1 *4` -> [4]

so with `*` the `aaa` is always an array even if you pass a single element. very useful

I didn't know `*` can also be used when calling methods.

Cheers!